pkg/flowkit/gateway/grpc.go

```diff
@@ -46,7 +47,7 @@ func NewGrpcGateway(host string) (*GrpcGateway, error) {
 
 	gClient, err := client.New(
 		host,
-		grpc.WithInsecure(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
```

Comment:
Thank you, this is good, but do you think we should actually make it secure by default? I was thinking to change that later. But this is a good patch anyhow.